src/Commands/Diagnostic.h

```diff
@@ -104,6 +104,7 @@ String Command_Debug(struct EventStruct *event, const char* Line)
 
 String Command_logentry(struct EventStruct *event, const char* Line)
 {
+	if (loglevelActiveFor(LOG_LEVEL_INFO)) addLog(LOG_LEVEL_INFO,Line);
```

Comment:
addLog is macro with test for LogLevel.... so this one is not needed another check.

Reply:
Right! I suffer from EOD (excessive optimization disease)... :)
Sent cleaned version